src/OpenTelemetry.AutoInstrumentation/Logging/LogLevel.cs

```diff
@@ -1,5 +1,9 @@
-// <copyright file="LogLevel.cs" company="OpenTelemetry Authors">
-// Copyright The OpenTelemetry Authors
+//------------------------------------------------------------------------------
```

Comment:
update license as it seems to originate from [here](https://github.com/serilog/serilog/blob/dev/src/Serilog/Events/LogEventLevel.cs)